src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -122,8 +122,7 @@ public async Task WriteTests(int[][] writes, WriteType writeType)
                         }
                     }
 
-                    stream.Shutdown();
-                    await stream.ShutdownCompleted();
```

Comment:
CompleteWrites will not wait for any completion, right? AFAIK we can't remove waiting for completion here and in the other tests. Both sides should receive SHUTDOWN_COMPLETED on app level before Connection.Close is called, otherwise we may get INVALID_STATE exceptions from msquic, because Connection.Close is happening immediately without waiting for anything.

Reply:
This will work because `stream.DisposeAsync()` now calls shutdown and waits for peer.